src/matrix/room/timeline/entries/BaseEventEntry.js

```diff
@@ -151,6 +155,58 @@ export class BaseEventEntry extends BaseEntry {
         return createAnnotation(this.id, key);
     }
 
+    _fallbackBlurb() {
+        switch (this.content.msgtype) {
+            case "m.file":
+                return "sent a file.";
+            case "m.image":
+                return "sent an image.";
+            case "m.video":
+                return "sent a video.";
+            case "m.audio":
+                return "sent an audio file.";
+        }
```

Comment:
`default` doesn't seem appropriate here since even the blurbs like "sent a file" need to be wrapped in `<mx-reply>` etc. as well. This is why `_replyFormattedFallback` calls out to `fallbackBlurb` and then wraps it in the HTML.